src/ts/uimanager.ts

```diff
@@ -10,6 +10,20 @@ import {BrowserUtils} from './browserutils';
 import { TimelineMarker, UIConfig } from './uiconfig';
 import { PlayerAPI, PlayerEventCallback, PlayerEventBase, PlayerEvent, AdEvent, LinearAd } from 'bitmovin-player';
 import { VolumeController } from './volumecontroller';
+import { i18n, CustomVocabulary, BitmovinPlayerUiTranslations } from './localization/i18n';
+
+/**
+ * language: defines the preferred locale.
+ * fallbackLanguages[]: Array of languages which should be used as fallback languages if given word doesn't exist
+ * in the selected vocabulary (in the order they are given in the array.)
+ * browserLanguageDetection: (default: true) will  enable/disable auto-detection and selection of browsers preferred language
+ * translations: key-value pair of 'language' and 'vocabulary' definitions. this is where the user adds their custom languages/vocabularies
+ */
+export interface LocalizationConfig {
```

Comment:
Please write the documentation of the properties directly above the properties as we do in other places as well. A general documentation for the interface can be above the interface.

Docu is outdated too.

Reply:
still open ^